sponsorship_compassion/models/res_partner.py

```diff
@@ -42,7 +42,7 @@ class ResPartner(models.Model):
     has_sponsorships = fields.Boolean(
         compute='_compute_has_sponsorships', store=True)
     number_sponsorships = fields.Integer(
-        compute='_compute_has_sponsorships', store=True)
+        compute='_compute_has_sponsorships')

```

Comment:
BTW: You cannot modify the method here, because field `member_ids` does not exist in this module. You should inherit the method compute in the module `partner_compassion` instead. So I think you can delete the commit in this repo and import your changes in `compassion-switzerland`